server/backend/api/app/schemas.py

```diff
@@ -300,8 +296,14 @@ class IdimProxyIdirInfo(BaseModel):
     @staticmethod
     def from_api_json(json_dict):
         info = IdimProxyIdirInfo(
-            found=json_dict['found'],
-            userId=json_dict['userId'],
-            displayName=json_dict["displayName"]
+            found=json_dict["found"],
+            userId=json_dict["userId"],
+            displayName=json_dict["displayName"],
         )
-        return info
\ No newline at end of file
+        return info
+
+
+class GCNotifyEmailParam(BaseModel):
```

Comment:
This is the only change in this file, the rest are just done by the auto formatter